app/views/works/_work_header_navigation.html.erb

```diff
@@ -1,6 +1,9 @@
 <!-- BEGIN navigation -->
 <h3 class="landmark heading"><%= ts("Actions") %></h3>
 <ul class="work navigation actions" role="menu">
+  <% if ( logged_in_as_admin? || permit?("tag_wrangler") ) %>
+    <li class="reindex"><%= link_to ts("Reindex Work"), reindex_work_path(@work) %></li>
+  <% end %>
```

Comment:
For readability, can you add a blank line between this and `<% if is_author_of?(@work) %>`?